taskcluster/ac_taskgraph/target_tasks.py

```diff
@@ -47,3 +50,23 @@ def filter(task, parameters):
         return task.attributes.get("build-type", "") == "release"
 
     return [l for l, t in full_task_graph.tasks.items() if filter(t, parameters)]
+
+
+@_target_task("default")
+def target_tasks_ac_default(full_task_graph, parameters, graph_config):
+    def filter(task):
```

Comment:
Note to self: Nice, I see we don't use `parameters` in the function signature anymore, in gecko taskgraph[1]. I like that! 

We could apply that in the Android monorepo too.

[1] For instance: https://searchfox.org/mozilla-central/rev/6f9fdc5c3b164a46004c98a5baaf55b05e2ad329/taskcluster/gecko_taskgraph/target_tasks.py#739